pkg/nfd-topology-updater/kubeletnotifier/kubeletnotifier.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubeletnotifier
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/klog/v2"
+
+	"github.com/fsnotify/fsnotify"
+)
+
+type EventType string
+
+const (
+	IntervalBased EventType = "intervalBased"
+	FSUpdate      EventType = "fsUpdate"
+)
+
+// TODO make path configurable
+const (
+	CPUStateFile     = "cpu_manager_state"
+	MemoryStateFile  = "memory_manager_state"
+	DevicesStateFile = "kubelet_internal_checkpoint"
+)
```

Comment:
you can also use `sets` package:
```go
var stateFiles = sets.NewString(
	"cpu_manager_state",
	"memory_manager_state",
	"kubelet_internal_checkpoint",
)
```
and then in line 80 simplify check to:
```go
if stateFiles.Has(e.Name) {
	i := Info{Event: FSUpdate}
	n.dest <- i
```

Reply:
+1 I liked the idea. 
Done, Thanks!